src/chat/chatAgentRequestHandler.ts

```diff
@@ -108,12 +110,20 @@ export const createSqlAgentRequestHandler = (
                 return { metadata: { command: "" } };
             }
 
+            const activity = startActivity(
```

Comment:
The `startConversation` here is an STS call and represents some 'internal' conversation. For tracking GHCP conversation, we should add telemetry at the top of the handler.